pkg/mls/store/queries.sql

```diff
@@ -31,11 +31,26 @@ INNER JOIN (
         address
 ) b ON a.address = b.address AND a.association_sequence_id = b.max_association_sequence_id;
 
+-- name: InsertAddressLog :one
+INSERT INTO address_log (address, inbox_id, association_sequence_id, revocation_sequence_id)
+VALUES ($1, $2, $3, $4)
+RETURNING *;
+
 -- name: InsertInboxLog :one
 INSERT INTO inbox_log (inbox_id, server_timestamp_ns, identity_update_proto)
 VALUES ($1, $2, $3)
 RETURNING sequence_id;
 
+-- name: RevokeAddressFromLog :exec
+UPDATE address_log
```

Comment:
Neat SQL here.

If this proves to be a pain, we could always just fetch the row using one query and then update it in another, filtering on the `association_sequence_id` from the first result.

But I like this more assuming it works correctly.

Reply:
@neekolas and @insipx, how do we feel about enforcing that there can only be one row for each `(address, inbox_id)` tuple, via a uniqueness/primary key constraint? Then the associate and revoke operations can both do an `insert ... on conflict update` operation, updating only if the new sequence ID is higher than the existing one.

Reply:
@insipx what do you think? I don't see any blockers.

And it's been weirding me out having the `address_log` with no primary key.